tungsten/__init__.py

```diff
@@ -1,6 +1,12 @@
-from tungsten.parsers.globally_harmonized_system.safety_data_sheet import \
+import os
+
+from tungsten.parsers.globally_harmonized_system.safety_data_sheet import (
     GhsSdsJsonEncoder
-from tungsten.parsers.supplier.sigma_aldrich.sds_parser import \
+)
+from tungsten.parsers.supplier.sigma_aldrich.sds_parser import (
     SigmaAldrichSdsParser
+)
 
 __all__ = ("GhsSdsJsonEncoder", "SigmaAldrichSdsParser")
+
+os.environ["TABULA_JAR"] = "tabula-1.0.6-SNAPSHOT-jar-with-dependencies.jar"
```

Comment:
Relative paths probably won't work here.

Something like this probably will:
```suggestion
os.environ["TABULA_JAR"] = str(Pathlib(__file__).parent / "tabula-1.0.6-SNAPSHOT-jar-with-dependencies.jar").resolve())
```

Reply:
Implemented in `[057e202](https://github.com/Den4200/tungsten/pull/13/commits/057e2020419da70bef5062db46e274d4e5dff3cd)`